src/aws_client.erl

```diff
@@ -26,7 +26,7 @@
 -type proto() :: binary().
 -type service() :: binary().
 -type endpoint() :: binary().
--type aws_client() :: map().
+-opaque aws_client() :: map().
 
 -export_type([access_key_id/0, secret_access_key/0, region/0,
               token/0, aws_client/0]).
```

Comment:
Aren't opaque types atomically exported?

Reply:
I get a warning if I remove the export: `opaque type aws_client() is not exported`